src/zzz_od/application/intel_board/intel_board_app.py

```
@@ -44,7 +44,16 @@ def __init__(self, ctx: ZContext):
         self.current_commission_type: CommissionType | None = None
         self.has_filtered: bool = False
 
-    @operation_node(name='返回录像店', is_start_node=True)
+    @operation_node(name='初始化加载', is_start_node=True)
+    def init_for_intel_board(self) -> OperationRoundResult:
+        try:
+            self.ctx.lost_void.init_lost_void_det_model()
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)